app/serializers/user_serializer.rb

```diff
@@ -30,4 +30,8 @@ def language
   def avatar
     user_avatar(object)
   end
+
+  delegate :confirm_terms, to: :object
+
+  delegate :email_notifs, to: :object
```

Comment:
I don't think these are necessary - the serializer should be able to do this automatically (as it does with the other values (ie name, email)

Reply:
I just tested yeah your right its not needed and works fine without. removed!